src/minErr.js

```diff
@@ -34,6 +34,7 @@ function minErr(module, ErrorConstructor) {
   ErrorConstructor = ErrorConstructor || Error;
   return function() {
     var SKIP_INDEXES = 2;
+    var MAX_DEPTH = 1;
```

Comment:
I don't think it'll be useful in this case. See https://github.com/angular/angular.js/pull/15433/files#r97893306.